src/main/java/com/marklogic/mule/extension/AbstractPagingProvider.java

```diff
@@ -38,7 +36,7 @@ protected AbstractPagingProvider(QueryParameters params) {
     protected final void handlePage(DatabaseClient databaseClient, Consumer<DocumentRecord> documentHandler) {
         currentPage++;
 
-        DocumentManager<JSONReadHandle, JSONWriteHandle> documentManager = databaseClient.newJSONDocumentManager();
+        GenericDocumentManager documentManager = databaseClient.newDocumentManager();
```

Comment:
@BillFarber  I changed this to GenericDocumentManager as I thought it was confusing to see these JSON handles when the code can return documents of any type.